core/runtime/src/main/java/io/quarkus/runtime/configuration/ConfigUtils.java

```diff
@@ -311,6 +312,11 @@ public static <T> Optional<T> getFirstOptionalValue(List<String> propertyNames,
         return Optional.empty();
     }
 
+    private static UUID generateUUID() {
+        Random random = new Random();
+        return new UUID(random.nextLong(), random.nextLong());
```

Comment:
Technically this might lead to an invalid UUID, since some bytes are reserved for the version, variant, etc... I don't know how critical that would be.

You should be able to create your own version of `java.util.UUID#randomUUID` that relies on a `Random`, though.